File: 1189_maximum-number-of-balloons/csharp/main.cs
using System;
using System.Text.Json;

public class Program
{
    public static void Main()
    {
        var lines = Console.In.ReadToEnd().Split('\n');
        string text = JsonSerializer.Deserialize<string>(lines[0]);
        var result = new Solution().MaxNumberOfBalloons(text);
        Console.WriteLine(JsonSerializer.Serialize(result));
    }
}

File: 1189_maximum-number-of-balloons/rust/main.rs
use std::io::{self, Read};

mod solution;
use solution::Solution;

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();
    let data: Vec<&str> = input.lines().collect();
    let text: String = serde_json::from_str(data[0]).unwrap();
    let result = Solution::max_number_of_balloons(text);
    println!("{}", serde_json::to_string(&result).unwrap());
}
